config/types.go

```diff
@@ -32,8 +37,24 @@ type FTWConfiguration struct {
 //	ForcePass is for tests you want to pass unconditionally. You should add a comment on why you force to pass the test
 //	ForceFail is for tests you want to fail unconditionally. You should add a comment on why you force to fail the test
 type FTWTestOverride struct {
-	Input     test.Input        `koanf:"input"`
-	Ignore    map[string]string `koanf:"ignore"`
-	ForcePass map[string]string `koanf:"forcepass"`
-	ForceFail map[string]string `koanf:"forcefail"`
+	Input     test.Input            `koanf:"input"`
+	Ignore    map[*FTWRegexp]string `koanf:"ignore"`
+	ForcePass map[*FTWRegexp]string `koanf:"forcepass"`
+	ForceFail map[*FTWRegexp]string `koanf:"forcefail"`
+}
+
+type FTWRegexp regexp.Regexp
+
+func (r *FTWRegexp) UnmarshalText(b []byte) error {
+	fmt.Println("Unmarshalling!")
```

Comment:
Use the logger. I don't think we want to see this message all the time. I also think the text could be more helpful. Maybe just remove the line entirely.

Reply:
Thanks, my bad, just a debugging print, not intended to be here.